src/pages/AMM/Pools/components/WithdrawForm/WithdrawForm.tsx

```diff
@@ -24,94 +52,125 @@ type WithdrawFormData = {
 
 type WithdrawFormProps = {
   pool: LiquidityPool;
-  onChangePool?: () => void;
+  slippageModalRef: RefObject<HTMLDivElement>;
+  onWithdraw?: () => void;
 };
 
-const WithdrawForm = ({ pool }: WithdrawFormProps): JSX.Element => {
+const WithdrawForm = ({ pool, slippageModalRef, onWithdraw }: WithdrawFormProps): JSX.Element => {
+  const accountId = useAccountId();
   const { t } = useTranslation();
   const prices = useGetPrices();
+  const { getAvailableBalance, getBalance } = useGetBalances();
+
+  const [slippage, setSlippage] = useState<number>(0.1);
```

Comment:
Same as above—unless there's a good reason to, we should put instance of `useState` above hook calls.